athena-clickhouse/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>aws-athena-query-federation</artifactId>
+        <groupId>com.amazonaws</groupId>
+        <version>2022.47.1</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>athena-clickhouse</artifactId>
+    <version>2024.10.1</version>
```

Comment:
The version number should be 2022.47.1 to be in line with all other connectors

Reply:
The same goes for the dependencies on other connectors. ex: [athena-cloudera-impala](https://github.com/awslabs/aws-athena-query-federation/blob/master/athena-cloudera-impala/pom.xml)

Reply:
Corrected now.  Thanks.